packages/admin-ui/src/stack/stack.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ComponentPropsWithoutRef, Ref } from 'react'
+import React, { forwardRef } from 'react'
+import type { ResponsiveProp } from '@vtex/admin-ui-react'
+import { useBreakpoint, getResponsiveValue } from '@vtex/admin-ui-react'
+import type { CSSPropAutocomplete, SpaceTokens } from '@vtex/admin-ui-core'
+import { cx } from '@vtex/admin-ui-core'
+import { stackTheme } from './stack.css'
+
+/**
+ * Component used to display a set of components that are spaced evenly.
+ * @example
+ * <Stack space="$space-3">
+ *  <Button />
+ *  <Button />
+ * </Stack>
+ */
+export const Stack = forwardRef(function Stack(
+  props: StackProps,
+  ref: Ref<HTMLDivElement>
+) {
+  const {
+    direction = 'column',
+    fluid = false,
+    align = 'start',
+    space = '$space-1',
+    className = '',
+    ...htmlProps
+  } = props
+
+  const { breakpoint } = useBreakpoint()
+
+  const responsiveDirection = getResponsiveValue(direction, breakpoint)
+  const responsiveFluid = getResponsiveValue(fluid, breakpoint)
+  const responsiveAlign = getResponsiveValue(align, breakpoint)
+  const responsiveSpace = getResponsiveValue(space, breakpoint)
+
+  return (
+    <div
+      ref={ref}
+      {...htmlProps}
+      data-direction={responsiveDirection}
+      data-fluid={responsiveFluid}
+      className={cx(stackTheme(responsiveAlign, responsiveSpace), className)}
+    />
+  )
+})
+
+Stack.displayName = 'Stack'
```

Comment:
# displayName

The displayName string is used in debugging messages. Usually, you don’t need to set it explicitly because it’s inferred from the name of the function or class that defines the component. You might want to set it explicitly if you want to display a different name for debugging purposes or when you create a higher-order component, see [Wrap the Display Name for Easy Debugging](https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging) for details.

https://reactjs.org/docs/react-component.html#displayname
```suggestion
```

Reply:
When creating the component using the `createComponent` function this was necessary, I forgot to remove it. Good catch!